store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreCache.java

```diff
@@ -120,7 +116,7 @@ public void addGraphToCache(final GraphSerialisable graphSerialisable, final byt
     public FederatedAccess getAccessFromCache(final String graphId) {
         try {
             final byte[] accessFromCache = cacheTransient.getAccessFromCache(graphId);
-            return (isNull(accessFromCache)) ? null : JSONSerialiser.deserialise(accessFromCache, FederatedAccess.class);
+            return (Objects.isNull(accessFromCache)) ? null : JSONSerialiser.deserialise(accessFromCache, FederatedAccess.class);
```

Comment:
It was already here and as I believe that this use of `Objects.isNull` is poor practice, `== null` should be used instead.